sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/monitoring/SdaMonitoringConfiguration.java

```diff
@@ -7,9 +7,28 @@
  */
 package org.sdase.commons.spring.boot.web.monitoring;
 
+import org.sdase.commons.spring.boot.web.auth.opa.OpenPolicyAgentHealthIndicator;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.autoconfigure.health.ConditionalOnEnabledHealthIndicator;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.PropertySource;
 
 @AutoConfiguration
 @PropertySource("classpath:/org/sdase/commons/spring/boot/web/monitoring/monitoring.properties")
-public class SdaMonitoringConfiguration {}
+public class SdaMonitoringConfiguration {
+
+  @Value("${opa.base.url:http://localhost:8181}")
+  private String opaBaseUrl;
+
+  @Value("${opa.policy.package:}")
+  private String policyPackage;
```

Comment:
To me it seems like this properties should not be in this place.

Is it possible to create the `OpenPolicyAgentHealthIndicator` as bean with the `OpaAccessDecisionVoter` as dependency and add a package private getter for `opaRequestUrl`? Then we don't even need to add it to this configuration.

Reply:
So if the AccessDecisionVoter is a dependency and it has the requestUrl and the opaRequestTemplate as well, I created a method there to call the get endpoint as package private, so I can use in my class as well